frontend/src/metabase/components/Banner/Banner.tsx

```diff
@@ -1,19 +1,49 @@
-import Markdown from "metabase/core/components/Markdown";
-import { Flex, type FlexProps } from "metabase/ui";
+import type { ReactNode } from "react";
 
-export const Banner = ({ children, ...props }: FlexProps) => {
-  const content =
-    typeof children === "string" ? <Markdown>{children}</Markdown> : children;
+import CS from "metabase/css/core/index.css";
+import type { FlexProps, IconName } from "metabase/ui";
+import { Flex, Group, Icon } from "metabase/ui";
 
+interface BaseBannerProps extends FlexProps {
+  icon?: IconName;
+  body: ReactNode;
+}
+
+type BannerProps =
+  | (BaseBannerProps & { closable: true; onClose: () => void })
+  | (BaseBannerProps & { closable?: false; onClose?: never });
+
+export const Banner = ({
+  icon,
+  body,
+  closable,
+  onClose,
+  bg,
+  ...flexProps
+}: BannerProps) => {
   return (
     <Flex
-      bg="bg-light"
-      c="text-medium"
       data-testid="app-banner"
-      p="0.75rem"
-      {...props}
+      align="center"
+      bg={bg || "bg-medium"}
+      py="sm"
+      justify="space-between"
+      pl="1.325rem"
```

Comment:
`1.325rem` seems quite specific here

Reply:
Yes, unfortunately we're doing silly things in the main nav sidebar.
it has its own padding, and then each menu item has its own padding, and then each icon in the menu item has... you guessed it  - its own padding.

So... 🤷‍♂️